_posts/0000-01-02-GDeelkens.md

```diff
@@ -0,0 +1,6 @@
+---
+lay-out: slide
+title: "Welkom to our second slide!"
+---
+Your text
```

Comment:
Dit is de eerste test met github
![image](https://user-images.githubusercontent.com/66379574/83696563-0789c600-a5fd-11ea-8ae7-254a247b32f4.png)

Reply:
`inser code  hahaha`

Reply:
hohoho